samples/secret.yaml

```diff
@@ -33,10 +33,35 @@
   # Default value: ""
   # This is an optional field from v2.10.0 onwards for PowerFlex storage system >=4.0.x
   nasName: "nas-server"
-# # To add more PowerFlex systems, uncomment the following lines and provide the required values
+  # zone is a cluster availability zone to which the PowerFlex system should be bound.
+  # The mapping is one-to-one - the PowerFlex system cannot belong to more than one zone.
+  # Ideally, the PowerFlex system and cluster nodes that define the availability zone would be
+  # geographically co-located.
+  # Optional: true
+  # Default value: none
+  zone:
+    # The name of the container orchestrator's availability zone to which the PowerFlex system
+    # should be mapped.
+    name: "zoneA"
+    # A list of protection domains to be utilized by PowerFlex system with optional pool assignment.
```

Comment:
BTW, the way we have it today is protection domain is optional but pool is mandatory. PD is only required when there is a conflict in pool names. 2 PDs having the same pool name. If pool names are unique, then that is sufficient in itself.

Reply:
Ah I see the structure now.
There can be more than one PD. Each PD can have many pools. If the pool names are not unique, we need the PD name as well.

Let me make some changes.